examples/spouse/deepdive.conf

```diff
@@ -0,0 +1,2 @@
+deepdive.calibration.holdout_fraction:0.6
```

Comment:
We started by testing a very small corpus and had that so that we could have a sufficient number of "spouse" instances in the test data. This was without using --sample_evidence.